transmogrifier/sources/json/aardvark.py

```diff
@@ -349,17 +349,41 @@ def get_publication_information(source_record: dict) -> list[str]:
         return publication_information
 
     @staticmethod
-    def get_rights(source_record: dict) -> list[timdex.Rights]:
+    def get_rights(source: str, source_record: dict) -> list[timdex.Rights]:
         """Get values from source record for TIMDEX rights field."""
         rights = []
 
         if "dct_accessRights_s" in source_record:
+            kind_access_rights = "Access"
+            kind_access_to_files = "Access to files"
             rights.append(
                 timdex.Rights(
-                    description=source_record["dct_accessRights_s"], kind="Access"
+                    description=source_record["dct_accessRights_s"],
+                    kind=kind_access_rights,
                 )
             )
 
+            if source == "gisogm":
+                rights.append(
+                    timdex.Rights(
+                        description="Not owned by MIT", kind=kind_access_to_files
```

Comment:
Curious to get GIS staff feedback on phrasing, I don't know what is expected here (or if it only matters as a facet)

Reply:
I think it is likely that we may have `MITAardvark` and `OGMAardvark` classes since we hardcoded a `gismit` string into `MITAardvark.get_source_link()` but we may find a better solution that avoids that